### ISSUE TITLE
Websocket SSL proxy setup for remote nodes

### ISSUE THREAD

**Issue by Swader** [w3f/polkadot-wiki#242]
Write up the process of configuring a remote node for JS access via polkadot.js.org.

TL;DR is:

```conf
server {

        server_name node.dotleap.com;

        root /var/www/node;
        index index.html;

        location / {
                try_files $uri $uri/ =404;

                proxy_pass http://localhost:9944;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header Host $host;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;

                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";
        }

    listen [::]:443 ssl ipv6only=on; # managed by Certbot
    # ...
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

```

**Comment by Swader** [comment]
Additionally, make a note that one should **never** expose a validator node like this, only nodes that serve as back end for something and have external auth exclusively.

**Comment by Swader** [comment]
Closed via #246
